appsec/tests/integration/src/test/groovy/com/datadog/appsec/php/integration/CommonTests.groovy

```diff
@@ -238,6 +238,20 @@ trait CommonTests {
         assert span.meta."_dd.appsec.event_rules.version" != ''
     }
 
+    @Test
+    void 'user login fingerprint'() {
+        def trace = container.traceFromRequest('/user_login_success.php?id=user2020') { HttpResponse<InputStream> resp ->
+            assert resp.statusCode() == 403
+            assert resp.body().text.contains('blocked')
+        }
+
+        Span span = trace.first()
+        assert span.meta."_dd.appsec.fp.http.endpoint" ==~ /^"http-get(-[a-zA-Z0-9]*){3}"$/
```

Comment:
Yeah I found this strange as well, but they are actually included in the value

Reply:
That's not right, it must be a side-effect of https://github.com/DataDog/dd-trace-php/blob/f8f3c99a0d6948d53a55efd1a2a65ec73b3e3901/appsec/src/helper/subscriber/waf.cpp#L239

Reply:
Indeed that's because of the `parameter_to_json` function, do you know why it was used here and if I can remove it  ?

Reply:
It is probably because schemas need to be converted to json, you could probably skip calling `parameter_to_json` based on whether the address is a schema or a fingerprint.